sql/python/sqlflow_submitter/db.py

```diff
@@ -55,6 +56,64 @@ def execute(driver, conn, statement):
     return field_names, field_columns
 
 
+def db_generator(driver, conn, statement,
```

Comment:
Would you mind adding tests to these two functions? :)

Reply:
Done.